tests/phpunit/tests/Story_Post_Type.php

```diff
@@ -285,8 +285,31 @@ public function test_add_caps_to_roles() {
 		$editor        = get_role( 'editor' );
 
 		foreach ( $all_capabilities as $cap ) {
-			$administrator->has_cap( $cap );
-			$editor->has_cap( $cap );
+			$this->assertTrue( $administrator->has_cap( $cap ) );
+			$this->assertTrue( $editor->has_cap( $cap ) );
```

Comment:
@spacedmonkey FYI, added some missing assertions here. Unfortunately didn't catch them before, but luckily the tests pass just fine still.